imports/plugins/core/product/server/no-meteor/utils/executeBulkOperation.js

```diff
@@ -0,0 +1,31 @@
+import Logger from "@reactioncommerce/logger";
+
+const logCtx = { name: "core/product", file: " bulkRemoveTagsFromProducts" };
```

Comment:
Might not be `bulkRemoveTagsFromProducts`

Reply:
added correct file name